actioncable/test/client_test.rb

```diff
@@ -98,7 +105,26 @@ def with_puma_server(rack_app = ActionCable.server, port = 3099)
       end
     end
   end
+  if defined?(::Iodine)
+    def with_iodine_server(rack_app = ActionCable.server, port = 3099, block)
+      ::Iodine.listen2http(app: rack_app, port: port.to_s, address: "127.0.0.1")
+      ::Iodine.workers = 1 # don't cluster the test
+      ::Iodine.threads = 1 # one for the server another for the task
+      t = Thread.new { ::Iodine.start }
+      block.call(port)
+      ::Iodine.stop
+      t.join
+    end
+  else
+    def with_iodine_server(rack_app = ActionCable.server, port = 3099, block)
+      puts "Iodine testing skipped, unsupported?."
```

Comment:
I think it's better to know a test wasn't performed rather than to have a silent behavioral change.

But if you prefer I just use `skip` (rather than maybe both), we can do that.

Reply:
Another possibility is doing separate test runs for each server, distinguished by an environment variable. Then we could run CI against MRI + Iodine in particular.

Reply:
If you prefer that I edit the CI testing as well, that's fine. I was thinking I should minimize the PR's effects on the project and automate what I can.

We can definitely remove the output and leave the tests to MRI alone. After all, if the code is broken, it will show on the MRI test and we don't need to repeat it to know its broken.